src/index.ts

```diff
@@ -1,7 +1,19 @@
 import { Pool, PoolClient } from 'pg'
 import { Migrator } from './migrator'
+
 const UniqueIndexViolationErrCode = '23505'
-const pool = new Pool({ connectionString: process.env.DATABASE_URL })
+
+const DefaultTimeout = 20_000
+const DefaultPoolSize = 10
+
+const pool = new Pool({
+  connectionString: process.env.DATABASE_URL,
+  connectionTimeoutMillis: Number(process.env.DATABASE_CONN_TIMEOUT ?? DefaultTimeout),
```

Comment:
CONN is not a typical acronym, I suggest to make it full (CONNECTION).

Reply:
Fixed